File: src/fs/ifelse.fs
let mutable value = true

if value
then printfn "Got true"
else printfn "Got else"

value <- false

if value 
then printfn "Got true" 
else printfn "Got else"

let result = 3
if result = 1
then printfn "Got true"
elif result = 2
then printfn "Got else"
else printfn "Got if else"

File: src/php/ifelse.php
<?php
$value = true;
if ($value) {
    echo "Got true\n";
} else {
    echo "Got else\n";
}

$value = false;
if ($value)
    echo "Got true\n";
else
    echo "Got else\n";

$result = 3;
if ($result == 1)
    echo "Got true";
elseif ($result == 2)
    echo "Got else";
else
    echo "Got if else";
?>